transport/src/main/java/io/netty5/channel/ChannelOutboundBuffer.java

```diff
@@ -90,29 +88,39 @@ protected BufferCache initialValue() {
 
     private boolean inFail;
 
+    // TODO: Does this need to be atomic / volatile?
```

Comment:
Generally speaking I think we could also just make this a normal volatile as we only increment from the `EventLoop` but need to read from any thread.